README.md

```diff
@@ -1,3 +1,6 @@
+![Solidus Starter Frontend Rails Storefront](https://user-images.githubusercontent.com/167946/216650338-6571185e-e88f-41dd-ba45-3c2907547aaa.png)
```

Comment:
Nit, but shouldn't it be safer to own it as part of the repo?

Reply:
We could, but I didn't want to think about where to store it in the repo. 😬  

So I thought: what can go wrong with using GH assets? And I couldn't find a valid answer to dissuade me.